awsemo/spiders/awsemo_spider.py

```diff
@@ -0,0 +1,10 @@
+import scrapy
+
+
+class AwsemoSpider(scrapy.Spider):
+    name = "awsemo"
+    start_urls = ['https://github.com/sindresorhus/awesome']
+
+    def parse(self, response):
+        for link in response.css('div.readme a::attr(href)').extract():
```

Comment:
`link` name is a bit misleading because we don't iterate over links (`a` tags) but over URLs.